docs/gitrepo-structure.md

```diff
@@ -207,3 +207,40 @@ will target `deployment.yaml`.  The patch will be applied using JSON Merge, Stra
 Which strategy is used is based on the file content. Even though JSON strategies are used, the files can be written
 using YAML syntax.
  
+## Cluster and Bundle state
+
+Clusters and Bundles has different states in different phases when applying bundles.
+
+For Bundles:
+
+**Ready**: Bundles have been deployed and all resources are ready.
+
+**NotReady**: Bundles have been deployed and some resources are not ready.
+
+**WaitApplied**: Bundles have been synced from controlplane and downstream cluster, but are waiting to be deployed.
+
+**ErrApplied**: Bundles have been synced from controlplane and downstream cluster, but there are some errors when deploying the bundle.
+
+**OutOfSync**: Bundles have been synced from controlplane, but downstream agent hasn't synced the change yet.
+
+**Pending**: Bundles are being processed by controlplane.
+
+**Modified**: Bundles have been deployed and all resources are ready, but there are some resources modified from outside changes.
```

Comment:
What's the definition of outside changes? Any changes not made by the Fleet controller?

Can we say

> **Modified**: Bundles have been deployed and all resources are ready, but there are some changes that were not made by the Fleet controller.

Reply:
I think it is better to say?
> Bundles have been deployed and all resources are ready, but there are some changes that were not made by git repository.